doc/api/net.md

```diff
@@ -1713,6 +1713,9 @@ changes:
     **Default:** `false`.
   * `pauseOnConnect` {boolean} Indicates whether the socket should be
     paused on incoming connections. **Default:** `false`.
+  * `blocklist` {net.BlockList} `blocklist` can be used for disabling inbound
+    access to specific IP addresses, IP ranges, or IP subnets. The IP is the
+    peer address of the accepted TCP socket, such as browser, reverse proxy.
```

Comment:
```suggestion
    access to specific IP addresses, IP ranges, or IP subnets. This does not
    work if the server is behind a reverse proxy, NAT, etc. because the address
    checked against the blocklist is the address of the proxy, or the one
    specified by the NAT.
```

Reply:
Thanks !